pkg/kubelet/nodestatus/setters.go

```diff
@@ -467,3 +467,16 @@ func OutOfDiskCondition(nowFunc func() time.Time, // typically Kubelet.clock.Now
 		return nil
 	}
 }
+
+// VolumesInUse returns a Setter that updates the volumes in use on the node.
+func VolumesInUse(syncedFunc func() bool, // typically Kubelet.volumeManager.ReconcilerStatesHasBeenSynced
```

Comment:
would prefer to have an interface which is a subset of the volume manager interface here as well that includes these two functions.